frontend/src/components/molecules/VerseNumbering.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from "react";
+import { Stack, Text } from "@chakra-ui/react";
+import { SongDetailQuery_details } from "@graphqlTypes/SongDetailQuery";
+
+interface VerseNumberingProps {
+  song: SongDetailQuery_details;
+}
+
+const VerseNumbering: FC<VerseNumberingProps> = ({ song: { verses } }) => (
+  <Stack spacing={6} h="100%">
+    {verses.map((verse, i) => (
+      <>
+        <Text key={`verse${i}`} whiteSpace="pre-wrap">
+          {`${i + 1}. ${verse}`}
+        </Text>
+      </>
```

Comment:
Remove fragment tag. 

```suggestion
        <Text key={`verse${i}`} whiteSpace="pre-wrap">
          {`${i + 1}. ${verse}`}
        </Text>
```